Capsulas/06 - Strings/codigos/argumentos/argumentos_2.c

```diff
@@ -0,0 +1,31 @@
+// Instrucciones de uso:
+//
+// gcc argumentos_2.c -o argumentos_2
+// ./argumentos_2 HelloWorld! 32
```

Comment:
```suggestion
// ./argumentos_2 HelloWorld! 4
```

Con menos repeticiones es más fácil verificar manualmente que el programa es correcto.